Jenkinsfile

```diff
@@ -234,9 +239,16 @@ pipeline {
                                         'E20R95KPAKSWTG'
                                     )
 
+                                    def previewAwsCloudFrontDistroId = (
+                                        env.BRANCH_NAME == 'prod' ? '' :
+                                        env.BRANCH_NAME == 'nonprod' ? '' :
```

Comment:
Seeing you're updating the Jenkinsfile can you please add in the cloudfront distribution IDs for nonprod: 
research-hub-preview = E1V3EOI1YKYNGI
research-hub = E22KP4O00VK941